app/src/pages/explorer/sample/sample.js

```diff
@@ -163,6 +169,18 @@ export default {
     }
   },
 
+  computed: {
+    sampleId () {
+      return this.$route.params.label ?? ''
+    },
+    sampleYear () {
+      return this.$route.params.label.split('-').pop() ?? ''
+    },
+    sampleAuthor () {
+      return this.$route.params.label.split('-').slice(0, -1).pop() ?? ''
+    }
+  },

```

Comment:
A vue router trick is that instead of having to write out this.$route.params.label every time, you can just put `props: true,` inside of the route definition in router/module/explorer.js, and then set  `props: ['label'],` inside of sample.js (see datavoyager for an example of this using chartId as the prop)

Reply:
I like the idea but I find this.$route easier to read and follow instead of modifying the router file and passing params as props. It would be helpful in the case where there are multiple parameters to pass, thanks.

Reply:
Also, in line 24, `sampleId` doesn't need to be defined. Line can be removed and instead use `this.sampleId` in line 25